src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -287,18 +288,60 @@ public static string GetDataSourceNameFromGraphQLContext(IMiddlewareContext cont
             {
```

Comment:
NIT: GetDataSourceNameFromGraphQLContext  method's description  seems to be misleading

Reply:
fixed.